README.md

```diff
@@ -106,3 +106,25 @@ The manual process of creating a new version is to:
        ]
     }
     ```
+
+## Overview
+* This repository contains JSON Schema files for all the versions of AsyncAPI specification. You can link to https://json-schema.org/ for reference.
+* These JSON Schema files do not reflect 1:1 the specification and shouldn't be treated as specification itself but rather as a tool for things like validation e.t.c
+* These JSON Schema files should'nt be used as the only tool for validation of AsyncAPI documents because some rules described in the AsyncAPI specification are not described here.
+
+## Custom Validation Needs
+* The JSON Schema does not cover all validation cases, and if a user decides to validate AsyncAPI tools with only JSON Schema provided in this repo, they risk having the AsyncAPI documents not working in tandem with all AsyncAPI tools and this will affect validation.
+* It's recommended to use the custom validation tool provided at https://github.com/asyncapi/parser-js as we look on implementing a comprehensive validation tool.
+ 
+* The following additional custom validations need to be provided:
+    1. Validate if variables provided in the url property have corresponding variable object defined and if the example is correct
```

Comment:
this should be the bullet list as the order doesn't matter.

ordering by number should be used only if the order makes a difference